src/main/java/org/plumelib/bcelutil/StackMapUtils.java

```diff
@@ -101,6 +106,21 @@
    */
   protected @NonNegative int number_active_locals;
 
+  /** The start of a local variable's live range: the first instruction in the range. */
```

Comment:
I think the next 5 variables are used only by `fix_local_variable_table` (and methods that only it calls).  Is that correct?  That could be useful to document.

Another thing that I think would help me understand the code is to put all the code that is related only to `fix_local_variable_table` together in one part of the file, delimited by comments at beginning and end (or just at beginning if it's the last thing in the file).  Then these variables can appear at the beginning of that section rather than here at the top of the file where it is not as clear what part of the file they are related to.